src/set_theory/surreal/basic.lean

```diff
@@ -95,6 +95,20 @@ theorem numeric_rec {C : pgame → Prop}
 | ⟨l, r, L, R⟩ ⟨h, hl, hr⟩ :=
   H _ _ _ _ h hl hr (λ i, numeric_rec _ (hl i)) (λ i, numeric_rec _ (hr i))
 
+theorem relabelling.numeric_imp {x y : pgame} (r : x ≡r y) (ox : numeric x) : numeric y :=
```

Comment:
This name looks sufficiently unusual and hard to find that I'd consider just inlining this into the other lemma via `have` or `suffices`.

bors d+

Reply:
(nevermind, I missed the approvals above)